tables/hbo.utb

```diff
@@ -148,65 +97,98 @@ letter ׄ 456-5  # upper dot
+nofor always ן\ 1345-456-16  # final Nun with  Back slash
+nofor always ןְ 1345  # final Nun with sheva
+nofor always ן־ 1345-36  # final Nun with Mediq
+nofor always ן׃ 1345-256  # final Nun with sof passuk
+nofor always ן” 1345-356  # final Nun with right quote
+nofor always ף  124-0  # final Pe with  space
+nofor always ף! 124-235  # final Pe with  exclamation
+nofor always ף" 124-6-2356  # final Pe with quote
+nofor always ף) 124-5-345  # final Pe with  right parenthesis
+nofor always ף, 124-2  # final Pe with  comma
+nofor always ף. 124-256  # final Pe with  period
+nofor always ף/ 124-456-34  # final Pe with  slash
+nofor always ף: 124-25  # final Pe with  colon
+nofor always ף; 124-23  # final Pe with  semi colon
+nofor always ף? 124-56-236  # final Pe with  question
+nofor always ף\ 124-456-16  # final Pe with  back slash
+nofor always ףְ 124-14  # final Pe with furtive patah
+nofor always ף־ 124-36  # final Pe with Mediq
+nofor always ף׃ 124-256  # final Pe with sof passuk
+nofor always ף” 124-356  # final Pe with right quote
+nofor always ץ  2346-0  # final tsadi with space
+nofor always ץ! 2346-235  # final tsadi with  exclamation
+nofor always ץ" 2346-6-2356  # final tsadi with quote
+nofor always ץ) 2346-5-345  # final tsadi with  right parenthesis
+nofor always ץ, 2346-2  # final tsadi with  comma
+nofor always ץ. 2346-256  # final tsadi with  period
+nofor always ץ/ 2346-456-34  # final tsadi with  slash
+nofor always ץ: 2346-25  # final tsadi with  colon
+nofor always ץ; 2346-23  # final tsadi with  semi colon
+nofor always ץ? 2346-56-236  # final tsadi with  question
+nofor always ץ\ 2346-456-16  # final tsadi with  back slash
+nofor always ץְ 2346-3  # final tsadi with sheva
+nofor always ץ־ 2346-36  # final tsadi with Mediq
+nofor always ץ׃ 2346-256  # final tsadi with sof passuk
+nofor always ץ” 2346-356  # final tsadi with right quote
+
 # punctuation op code characters
 punctuation ֑ 23  # atnokta
 punctuation ־ 36  # maqaf
 punctuation ׀ 456  # paseq
 punctuation ׃ 256  # sof pasuq
 punctuation ״ 356  # gershayim
 
-# sign op code characters
-sign ✔ 12356  # Check Mark Distinguishes Same kind of Varient
-sign ✕ 1235  # Mark beginning of Replacement
-sign ✖ 2456  # Mark Ends of Replacement
-sign ✗ 24  # Mark for  Insertion
-sign ✘ 35  # Marks Varients with in Apparatus
-sign ✙ 23456  #   Marks end of Replaced text
-sign ❁ 123-1346-1346  # Reference to LXX
-sign ❍ 135  # Marks Omitted Word
-sign ❏ 135-3  # Marks End of Omited word
-sign ❐ 1235  # Marks Replaced Word
-sign ❑ 12345  # Mark that Distinguishes between Varient
-sign ❘ 1256  # Mark that separates variations
-sign ❙ 2345  # Mark that indicates Alternative readings
-sign ❚ 14  # Mark indicates varrnt
-sign ❥ 345  # Marks distinguishes between varient
-sign ❦ 126  # Marks beginning of transposition
-sign ❧ 156  # Check Marks end of transposition
-
-# Include additional braille tables
+include criticle-apparatus.uti
```

Comment:
I think this needs to go below `he-common-vowels-ihbc.uti` to avoid issues with back translation, at least when there's overlap in braille patterns.

Reply:
Fixed